users/user-validation.js

```diff
@@ -0,0 +1,16 @@
+function userValidation() {
```

Comment:
annnnnnnd..... you did have validation afterall lol.  But I don't see it being implemented?